vsts_cd_manager/continuous_delivery_manager.py

```diff
@@ -106,18 +119,13 @@ def setup_continuous_delivery(self, azure_deployment_slot, app_type, vsts_accoun
         account_created = False
         accountClient = Account('3.2-preview', None, self._azure_info.credentials)
         if create_account:
-            # Try to create the account (already existing accounts are fine too)
```

Comment:
Make sure the run the tests and consider adding some unit tests to cover this new code.